src/tracing.rs

```diff
@@ -67,6 +68,11 @@ pub enum Event<'a> {
 		is_static: bool,
 		context: &'a Context,
 	},
+	Log {
+		address: H160,
+		topics: Vec<H256>,
+		data: Vec<u8>,
```

Comment:
To be consistent with other events
```suggestion
		data: &'a [u8],
```

Reply:
Updated in https://github.com/moonbeam-foundation/evm/pull/4/commits/ba040f5a16a6e51f06610bc0eaf56bffde2c8a89